### PR TITLE
Added notes for using MyGet feed

### PR DESCRIPTION
**Pull request by MisterJames**

### CODE REVIEW

**Comment by NTaylorMullen** [comment]
Include a link to a nuget tutorial on adding sources.  Thanks for this PR!

**Comment by MisterJames** [comment]
@NTaylorMullen How does this look? Expanded the text a bit with links and a more thorough explanation, let me know if you'd like to see something else here. Cheers!

File: README.md [hunk 1]
```diff
@@ -17,3 +17,12 @@ Feel free to stop by and say hi at http://jabbr.net/#/rooms/ShootR.
 
 **SignalR Source**: https://github.com/SignalR/SignalR  
 **EndGate Source**: https://github.com/ntaylormullen/endgate  
+
+###Building For the First Time###
```
Comment (NTaylorMullen) [inline README.md]: for

File: README.md [hunk 2]
```diff
@@ -17,3 +17,12 @@ Feel free to stop by and say hi at http://jabbr.net/#/rooms/ShootR.
...
+The latest versions of Visual Studio are now cofigured to restore missing packages in solutions that you open, however, ShootR uses packages from a custom package source. 
+To restore packages in the ShootR solution, you'll need to make sure that your installation of Visual Studio is configured to [restore missing packages] (http://docs.nuget.org/docs/workflows/using-nuget-without-committing-packages). 
+
+You'll also need to add the nightly build feed from the Asp.Net web stack to your NuGet package sources, which is located here: http://www.myget.org/F/aspnetwebstacknightly/
```
Comment (NTaylorMullen) [inline README.md]: Period at end of sentence.

**Comment by NTaylorMullen** [comment]
Thanks for the PullRequests @MisterJames !

**Pull request merged**